pkg/kubelet/apis/kubeletconfig/v1alpha1/types.go

```diff
@@ -86,6 +86,9 @@ type KubeletConfiguration struct {
 	// tlsPrivateKeyFile is the ile containing x509 private key matching
 	// tlsCertFile.
 	TLSPrivateKeyFile string `json:"tlsPrivateKeyFile"`
+	// CipherSuites is the list of allowed cipher suites for the server.
+	// Values are from tls package constants (https://golang.org/pkg/crypto/tls/#pkg-constants).
+	CipherSuites []string `json:"tlsCipherSuites"`
```

Comment:
The json name should match the name on the config object, with the exception of the lowercase first letter. Please change the names on the config objects to `TLSCipherSuites`.